ConfigOptionData vs ConfigOption (#228)
replicatedhq/kots.io: content/vendor/packaging/private-images.md

When delivering a Kots application through vendor.replicated.com, there’s built-in support to include private images -- without managing or distributing actual registry credentials to your customer. The license file grants revokable image pull access to private images, whether these are stored in the Replicated private registry or another private registry server that you’ve decided to use.

> If your application images are already available in a private, but accessible image registry (such as Docker Hub, quay.io, ECR, GCR or such), then your application licenses can be configured to grant proxy, or pull-through, access to the assignee without giving actual credentials to the customer. This is useful and recommended because it prevents you from having to modify the process you use to build and push application images, and it gives you the ability to revoke a customer’s ability to pull (such as on trial expiration). This External Registry is shared across all KOTS applications in a team, this is so that the same images can be used across multiple apps.

Reviewer: ```suggestion
If your application images are already available in a private, but accessible image registry (such as Docker Hub, quay.io, ECR, GCR or such), then your application licenses can be configured to grant proxy, or pull-through, access to the assignee without giving actual credentials to the customer. This is useful and recommended because it prevents you from having to modify the process you use to build and push application images, and it gives you the ability to revoke a customer’s ability to pull (such as on trial expiration). This External Registry is shared across all KOTS applications in a team, allowing images to be used across multiple apps.
```
